tox.ini

```diff
@@ -64,7 +64,7 @@ commands_pre =
 changedir = {homedir}
 usedevelop = false
 commands =
-    pyargs: pytest {toxinidir}/docs --pyargs {posargs:jwst}
+    pyargs: pytest --report-crds-context {toxinidir}/docs --pyargs {posargs:jwst}
```

Comment:
Is it necessary to add this?  I think by including `{toxinidir}/docs` this job pulls in `setup.cfg`.  See a previous PR:

https://github.com/spacetelescope/jwst/runs/2661207806?check_suite_focus=true#step:7:22

Reply:
Oops, no, it's not.  I tried with and without that argument and got the header line both times.